docs/report/report.tex

```diff
@@ -189,174 +189,164 @@ \subsection{Deployment Cycle}
 \subsubsection{Retrospektive}
-\colorbox{red}{TODO:}Was haben wir dabei gelernt? Neue Erkentnisse? Neue Sichtweisen?
-Was lief gut, neu gelernt, was lief nicht so gut, was verbessern?
-\begin{comment}{
-
-Lief Gut
-- Zusammenarbeit und Team Environment
-- Meetings und Zusammenarbeit
-- Github Workflow
-- Anton!!!
-- Richtig viel gelernt.
-- Sau viel Zeit Einsatz von Anton, muss man mal wertschätzen.
-- gruppenklima
-- Die Kommunikation war klasse, genauso Teamwork.
-
-Neu Gelernt
-- Vue
-- Feathers
-- Tailwind
-- CI
-- undefined & null
-- feathers
-- vue
-- Code / PR Review
-
-Nicht so gut
-- globale Probleme waren meh
-- bei Problemen (nicht weitergekommen) nicht sofort bescheid sagen
-- Wörtlicher Kundenwunsch wurde viel zu spät in dem sprint behandelt. Auch wenn hello wolrd cool war müssen wir vielleicht in dem nächsten sprint härter priorisieren.
-
-Verbessern
-- Aufgabenaufteilung
-- Code reviews häufiger machen
-- Aufgaben früher
-- Teils lange Wartezeiten
-- mehr den aktuellen stand kommunizieren
-- Pünktlichkeit und rumorganisation der stand ups.
-}
+Nach dem ersten Meilenstein fand noch keine nennenswerte Retrospektive statt.
 
 \subsection{Meilenstein 2}
-\colorbox{red}{TODO:}Kurze Einleitung,  von wann bis wann ging}
+Der zweite Meilenstein wurde für den Zeitraum 11.11.2020 bis 25.11.2020 angelegt.
+Das Team sollte sich hier die ersten Grundbausteine für das Projekt legen.
 \subsubsection{Ziel}
-\colorbox{red}{TODO:}was haben wir uns vorgenommen, was war das ziel, was wollten wir schaffen?}
+Damit das Team ein generelles Verständnis des Technologie-Stacks entwickeln konnte, musste jedes Teammitglied eine \glqqHello-World\grqq Übungsaufgabe absolvieren. Diese beinhaltete im Backend einen Feathers-Service für ein virtuelles Haustier anzulegen und diese im Frontend darzustellen.
+Darüber hinaus sollte sich mit dem Raspberry Pi und dem NFC-Reader auseinandergesetzt werden, sodass NFC-Tags ausgelesen werden können.
+Damit die Entwicklung zeitnah starten konnte, sollten die ersten Mockups für das User Interface erstellt werden sowie die Grundstruktur für den Bericht zu dem Projekt angefertigt werden, damit dieser begleitend schon gefüllt werden konnte.
 \subsubsection{Probleme}
-\colorbox{red}{TODO:}welche prrobleme sind aufgetreten?}
-
+Zu Beginn ging der Kundenwunsch, dass bereits NFC-Tags ausgelesen und angezeigt werden können, unter und führte \colorbox{yellow}{zu Problemen mit der Zeit}.
+Abgesehen davon war nur die Einarbeitung in die genutzten Technologien für manche schwieriger.
 \subsubsection{Lösungen}
-\colorbox{red}{TODO:}Welche Lösungen haben wir gefunden?}
-
+Bereits von Anfang an zeigte sich ein sehr guter Zusammehalt im Team und bei Problemen wurde sich schnell untereinander im Team geholfen.
+Um den Kundenwunsch noch sinnvoll umzusetzen, wurden gegen Ende des Sprints Überstunden geleistet.
```

Comment:
```suggestion
Um das vom Kunden gewünschte Feature noch umzusetzen, wurden gegen Ende des Sprints Überstunden geleistet.
```

Reply:
umgesetzt in https://github.com/project-leek/project-leek/pull/409/commits/db7fedecbe3f12f759a2c504c4b9f0e5a7061fee